### ISSUE TITLE
Extrusion multipler for each filament

### ISSUE THREAD

**Issue by EdMakes** [MosaicManufacturing/canvas-issues#97]
Hello!

I have a few spools of a wonderful silky metallic PLA filament that is very reasonably priced. The issue is that this filament has a consistently undersized diameter by 3-5%. With a single-color print this is no problem as I can simply bump up the extrusion multiplier. With a multi-color print using a filament at the extreme edges of manufacturing tolerances, I have to balance over- and under-extrusion of the different diameter filaments.

I request that a feature be added to canvas to opt to set the extrusion multiplier individually for each filament. This would allow better tuning for pairing filaments that may be at the extreme ends of manufacturing tolerances.

In the meantime, I am looking into using Cura and Chroma to see if I can figure out a way to achieve this result.

Thank you for your time and consideration of this feature request.

**Comment by brandonbloch** [comment]
Hi there,

This is actually possible, by creating a material profile for this filament and assigning this to the input you'll be using it on with Palette. Note that you can adjust both the filament diameter (e.g. setting it to 1.70 mm) and/or the extrusion multiplier (e.g. setting it to 1.05 x)—these will have slightly different effects based on which calculations use them, but may both be worth playing with, either separately or in combination with one another.

![Screen Shot 2019-04-15 at 7 39 51 AM](https://user-images.githubusercontent.com/217964/56130162-29843d00-5f52-11e9-9036-eff3866c8338.png)

Basically if you assign a material with style setting overrides to an input, the overrides will take precedence for all models using that material. There are other overrides that you can enable, but I'm just mentioning the relevant one, extrusion multiplier. I hope that this does the trick for you!

Cheers,
Brandon
